tools/clang/lib/Sema/SemaHLSLDiagnoseTU.cpp

```diff
@@ -523,7 +523,7 @@ void hlsl::DiagnoseTranslationUnit(clang::Sema *self) {
     }
 
     DXIL::ShaderKind EntrySK = shaderModel->GetKind();
-    DXIL::NodeLaunchType NodeLaunchTy = DXIL::NodeLaunchType::Invalid;
+    DXIL::NodeLaunchType NodeLaunchTy = DXIL::NodeLaunchType::Broadcasting;
```

Comment:
I think a better fix is to set this to Broadcasting below if `EntrySK == DXIL::ShaderKind::Node` and no `HLSLNodeLaunchAttr` is found.  Only if this is actually a node shader should this be set to something other than `Invalid`.

Reply:
Updated.